akka-discovery/src/main/scala/akka/discovery/ServiceDiscovery.scala

```diff
@@ -53,10 +81,10 @@ object ServiceDiscovery {
    * @param port optional port number
    * @param address optional IP address of the target. This is used during cluster bootstap when available.
    */
-  final case class ResolvedTarget(
-    host:    String,
-    port:    Option[Int],
-    address: Option[InetAddress]
+  final class ResolvedTarget(
+    val host:    String,
+    val port:    Option[Int],
+    val address: Option[InetAddress]
```

Comment:
no Java getter for host

Reply:
um, I added getters for those that had different return types
(I'm not overly excited about sprinkling getters everywhere)